filebeat/module/mysql/slowlog/test/mysql-5.7.22.log-expected.json

```diff
@@ -0,0 +1,79 @@
+[
+    {
+        "@timestamp": "2018-08-07T08:27:47.000Z", 
```

Comment:
I worried at first that taking the unix timestamp instead of the title one we would loose the timezone diff. But seems like the timestamp is in UTC which is good.

Reply:
@ycombinator I remember you saw something different for ES for example?
@kaiyan-sheng Could you try to run mysql in a non UTC timezone environment and check if the timestamp is still correct. If not, we should do a follow up PR that adds convert_timezone.

Reply:
@ruflin I used to start test environment under beats/testing/environments using `make start`. Now instead of that, I started Elasticsearch locally on my macbook and ran `GENERATE=1 INTEGRATION_TESTS=1 TESTING_FILEBEAT_MODULES=mysql nosetests -v --nocapture tests/system/test_modules.py` again. I think this tested the `non UTC timezeon environment` scenario. Is that right?

Reply:
> I remember you saw something different for ES for example?

The ES (and LS) logs contain ISO8601 timestamps without the timezone. In MySQL the value of the [`timestamp` system variable](https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_timestamp) needs to be a Unix timestamp, which is in UTC already, so we're good here.

Reply:
Thanks for the details @ycombinator